symplyphysics/laws/kinematic/damped_oscillations/exponential_decay_constant.py

```diff
@@ -0,0 +1,47 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    angle_type,
+    dimensionless,
+    validate_input,
+    validate_output,
+    print_expression,
+)
+
+# Description
+## For any (positive) value of the damping ratio, the position of a damped oscillator
+## is subject to exponential decay, i.e. its position is proportional to exp(-lambda*t)
+## where lambda is the exponential decay constant and t is time.
+
+# Law: lambda = omega * zeta
```

Comment:
It seems, it is approximate and for underdamped case.

Reply:
What about this?

Reply:
Why do you think it is approximate? I don't fully understand what you mean by that word. The solution of the damped oscillator equation contain the term $`\exp(-\lambda t)`$, where $`\lambda = \zeta \omega_0`$ by design of that equation.

Reply:
My mistake. I've seen someone mentioned it, but it seems I got it wrong.